packages/utils/upgrade/resources/codemods/5.0.0/comment-out-lifecycle-files.code.ts

```diff
@@ -0,0 +1,62 @@
+import { Transform } from 'jscodeshift';
+
+/**
+ * comments out lifecycles.js/ts files and adds a description for the reason at the top
+ */
+const transform: Transform = (file, api) => {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  if (file.path.endsWith('lifecycles.js') || file.path.endsWith('lifecycles.ts')) {
```

Comment:
Do you think we could be a bit more specific/precise when targeting the files? What if apps have `lifecycles` files dedicated to other purposes? :thinking:

Reply:
sure I can make it look for `lifecycles` files that are under `content-types` directory only